app.js

```diff
@@ -1,93 +1,109 @@
 'use strict';
-var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
+
+function middleName() {
+  var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
   middleName.toLowerCase();
-if (middleName === 'no') {
+  if (middleName === 'no') {
     alert('Wow, you guessed that my middle name is not Charles. It\'s actually Clifford.');
     console.log(middleName);
-} else {
+  } else {
     alert('You are right, my middle name is actually Clifford.');
     console.log(middleName);
-}
+  }
+} middleName();
 
-var interests = prompt('Can you guess if I\'m interested in coding?');
-if (interests.toLowerCase() === 'yes') {
+function interests() {
+  var interests = prompt('Can you guess if I\'m interested in coding?');
+  if (interests.toLowerCase() === 'yes') {
     alert('Nice, you guessed my primary interest is coding.');
     console.log(interests);
-} else {
-    alert('Actually, my primary interest is coding.')
+  } else {
+    alert('Actually, my primary interest is coding.');
     console.log(interests);
-}
+  }
+} interests();
 
-var interestingFact = prompt('Do you think I attended more than 5 public schools thru High School?');
-if (interestingFact.toLowerCase() === 'yes') {
+function functionName() {
+  var interestingFact = prompt('Do you think I attended more than 5 public schools thru High School?');
+  if (interestingFact.toLowerCase() === 'yes') {
     alert('You are good! I did attend more than 5 schools. I actually attended 6 schools as a \'military brat\' growing up.');
     console.log(interestingFact);
-} else {
+  } else {
     alert('Nope, it was 6 public schools. I was a military brat and moved quite a lot growing up.');
     console.log(interestingFact);
-}
+  }
+} functionName();
 
-var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
-if (careerGoals.toLowerCase() === 'yes') {
+function careerGoals() {
+  var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
+  if (careerGoals.toLowerCase() === 'yes') {
     alert('You must have a crystal ball! Yep, being a Cartographer is in fact my career goal');
     console.log(careerGoals);
-} else {
-    alert('Well almost... my career goal is in fact to be a Cartographer.')
+  } else {
+    alert('Well almost... my career goal is in fact to be a Cartographer.');
     console.log(careerGoals);
-}
+  }
+} careerGoals();
 
-var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
-if (focus.toLowerCase() === 'yes') {
+function focus(){
+  var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
+  if (focus.toLowerCase() === 'yes') {
     alert('Impressive, you are on the money!');
     console.log(focus);
-} else {
+  } else {
     alert('Actually, I\'m focusing my study on Python, as it is apparently the current standard in the GIS industry');
     console.log(focus);
-}
+  }
+} focus();
```

Comment:
your function declarations should end with a semicolon, and your function call should be on its own line